### PR TITLE
 Added Golang Logic/Math Operators

### LINKED ISSUE #300: Add Go Logic/Math Operators
# Description

Logic/Math Operators for Go aren't presently on Code Thesaurus. This issue is to add that concept to the system.

# Requirements

* Copy `web/thesauruses/_meta/operators.json` to the right Go directory in `web/thesauruses/`
* Edit the operators.json file to accurately reflect the latest major version of Go
* Delete the `"name":` fields before your PR
* Add in `"comment":` if you wish to explain something in the code
* Add in `"not-implemented": "true"` if the concept doesn't exist in the language. You can then leave `"code":` blank.

# Claiming This Issue

Please leave a comment below so I can assign you to it. This is to prevent multiple people from working on the same issue. 

Check out [CONTRIBUTING.md](https://github.com/codethesaurus/codethesaur.us/blob/main/CONTRIBUTING.md) for more information on how to contribute. 

Check out [https://docs.codethesaur.us/thesaurus/](https://docs.codethesaur.us/thesaurus/) for information on how Code Thesaurus works and how to add to the thesauruses.

# Need Help?

Leave a comment on here with your question, or feel free to ask [@geekygirlsarah](https://twitter.com/geekygirlsarah) on Twitter. You can also email coreteam@codethesaur.us too.

# Thanks!

We really value and appreciate the additions people make to Code Thesaurus, no matter how large or small. So thank you for taking the time to work on this!

### PR DESCRIPTION
**Pull request by 26tanishabanik**
## What GitHub issue does this PR apply to?

Resolves #300 

## Checklist

- [x] I claimed any associated issue(s) and they are not someone else's
- [x] I have looked at documentation to ensure I made any revisions correctly
- [x] I tested my changes locally to ensure they work
- [ ] (If editing Django) I have added or edited any appropriate unit tests for my changes

### CODE REVIEW

File: web/thesauruses/go/operators.json [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+      "right_shift",
+      "right_shift_assignment"
+    ],
+    "Conditional Operators": [
+      "ternary"
+    ]
+  },
+  "operators": {
+    "addition": {
+      "name": "Addition operator",
```
Comment (geekygirlsarah) [inline web/thesauruses/go/operators.json:54]: You can remove all the `name` lines as the site doesn't pull text from the language files.

File: web/thesauruses/go/operators.json [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+      "name": "Increment (add 1) operator",
+      "code": "op++"
+    },
+    "decrement": {
+      "name": "Decrement (subtract 1) operator",
+      "code": "op--"
+    },
+    "exponential": {
+      "not-implemented": "true",
+      "code": ""
```
Comment (geekygirlsarah) [inline web/thesauruses/go/operators.json:103]: You can remove `code` lines from any of the `not-implemented: true` concepts. It just makes it a bit cleaner.

File: web/thesauruses/go/operators.json [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+    },
+    "receive_operator": {
+      "name": "Receive Operator",
+      "code": "var <- ch"
+    },
+    "short_variable_declaration": {
+      "name": "Short Variable Declaration",
+      "code": "var := val"
+    },
```
Comment (geekygirlsarah) [inline web/thesauruses/go/operators.json:124]: I'm pretty sure this isn't in the meta `operators.json` file. You can't really add in new things like this. If you think things are missing, you're welcome to add an issue to have someone add them, but they don't really work on a single language like this.

Can you remove these or any other ones you added in?

**Review by geekygirlsarah — CHANGES_REQUESTED** [review]
I think with the tweaks mentioned in here, I'd be happy to merge it in!

File: web/thesauruses/go/operators.json (hunk 3 above)
Comment (26tanishabanik) [reply web/thesauruses/go/operators.json:124]: Sure, I will strictly follow the operators.json, also do you want me to remove the "name" tag from all the objects?

**Comment by geekygirlsarah** [comment]
Hey @26tanishabanik! It's been about two weeks since you made this PR and I suggested some changes. Is it still going ok? Do you have any questions?

**Comment by 26tanishabanik** [comment]
> Hey @26tanishabanik! It's been about two weeks since you made this PR and I suggested some changes. Is it still going ok? Do you have any questions?

I have asked you a question above, can you please answer that first?

**Comment by geekygirlsarah** [comment]
> I have asked you a question above, can you please answer that first?

I missed that. It would have been faster for you to re-ask it or quote it or something.

File: web/thesauruses/go/operators.json (hunk 3 above)
Comment (geekygirlsarah) [reply web/thesauruses/go/operators.json:124]: Yes, remove `name` since that's in the meta language file.

**Comment by geekygirlsarah** [comment]
Hi @26tanishabanik! It's been about a week since my response to your question. How is this coming along? Do you have any questions?

**Comment by 26tanishabanik** [comment]
> Hi @26tanishabanik! It's been about a week since my response to your question. How is this coming along? Do you have any questions?

I am really sorry, I have been busy with my academic assignments, I will surely update it by tomorrow. My sincere apologies for the delay.

**Comment by geekygirlsarah** [comment]
> 
> I am really sorry, I have been busy with my academic assignments, I will surely update it by tomorrow. My sincere apologies for the delay.

That's alright. I just wanted to make sure you weren't stuck and if you wanted to get it done in time for Hacktoberfest it wasn't me holding you up.

**Comment by geekygirlsarah** [comment]
Hey @26tanishabanik, how is this coming along? Do you have any questions or anything you need from me to finish this PR?

**Comment by 26tanishabanik** [comment]
> Hey @26tanishabanik, how is this coming along? Do you have any questions or anything you need from me to finish this PR?

Hey @geekygirlsarah sorry for the delay, its done, PR no #452

**Comment by geekygirlsarah** [comment]
Oh... I was expecting you to make the changes to this PR. You can always keep adding more commits to the same branch and they'll always get automatically added into the same PR.

~~Just to confirm... should this PR be closed in favor of #452 instead?~~ Never mind, I already merged the other one. I'll close this then.

**Comment by 26tanishabanik** [comment]
> Oh... I was expecting you to make the changes to this PR. You can always keep adding more commits to the same branch and they'll always get automatically added into the same PR.
> 
> ~Just to confirm... should this PR be closed in favor of #452 instead?~ Never mind, I already merged the other one. I'll close this then.

Sure

**Pull request closed without merging**